standard_renderer.go

```diff
@@ -34,6 +34,7 @@ type standardRenderer struct {
 	ticker             *time.Ticker
 	done               chan struct{}
 	lastRender         string
+	lastRenderedLines  []string
 	linesRendered      int
```

Comment:
We can possibly remove the `linesRendered` field and use `len(lastRenderedLines)` instead. What do you think?

Reply:
It will break `tea.Println`. We reset the `lastRenderedLines` when we call `repaint`, which is the case when using `tea.Println`. We need to keep track of the number of lines last rendered for `tea.Println` to work.